src/Game/Models/Board.php

```diff
@@ -0,0 +1,61 @@
+<?php
+
+namespace Game\Models;
+use Game\BoardInterface;
+/**
+ * Class Board
+ * @package Game\Models
+ */
+class Board implements BoardInterface
+{
+    const ZERO_CONST = 0;
+
+    /** @var  integer $size */
+    protected $size;
+
+    /** @var array $area */
+    protected $area = array();
+    
+    /**
+     * @param integer $size
+     */
+    public function setAreaSize($size)
+    {
+        $this->size = $size;
+    }
+    
+    /**
+     * @return integer
+     */
+    public function getAreaSize()
+    {
+        return $this->size;
+    }
+    
+    /**
+     * @return array $this->area
+     */
+    public function setBoard()
+    {
+        for ($x = 1; $x <= self::getAreaSize(); $x++) {
```

Comment:
does it work?
You are trying to call non-static method getAreaSize like it is static one?
Probably you need $this->getAreaSize()?
Otherwise you should have static function getAreaSize - therefore all boards will have the same size - which is not useful

Reply:
Yes sir!